### ISSUE TITLE
Studio: Further enhancements and restructuring of the contents

### ISSUE THREAD

**Issue by tapadipti** [iterative/dvc.org#2515]
- [x] Add more instructions/screenshots for GH app installation
_Originally posted by @tapadipti in https://github.com/iterative/dvc.org/pull/2510#discussion_r642501595_

 - [x] Remove _v2 from image filenames.
 https://github.com/iterative/dvc.org/pull/2510#discussion_r643136007
 
  - [x] Fix the banner image properties if needed
 https://github.com/iterative/dvc.org/pull/2510#discussion_r643186606

  - [x] Define roles of collaborators in a tabular format
  https://github.com/iterative/dvc.org/pull/2510#discussion_r643163276

  - [x] In View Settings document, explain DVC or link to relevant doc
  https://github.com/iterative/dvc.org/pull/2510#discussion_r643171177

 - [x] Add section on "Troubleshooting"
https://github.com/iterative/dvc.org/pull/2510#discussion_r643636495

**Update on 2021-06-14: Added the following items from [this doc](https://docs.google.com/document/d/1u9favvPTKdgZbbJhpRpGsN5x6ft9ZRiYnNaxSNztTwM/edit)**

**Index page:**
 - [x] Add a video about What is DVC Studio. Remove the screenshot.
 - [x] Revise the list of things that you can use Studio to do.
 - [x] Add a section outlining the rest of the tutorial like in https://dvc.org/doc

**Get Started page:**
 - [x] Add an Outline section, which lists the basic steps to get started. Eg:

- Prepare your repo
- Create a view. The DVC Studio Views Page section should be within this step.
- Generate plots
- Run experiments
- Etc.

Each item in this list should link to corresponding sections in the documentation
 - [x] Preparing your repositories: Elaborate this more
 - [x] Change the first screenshot to show the welcome banner and the demo view that we should when someone first logs in
 - [x] Components of a view: In the screenshot, label the different components. You can use numbers as labels (eg, 1, 2, ..) and in the text below the screenshot, write which number corresponds to what. Eg, 1: Filters. 2: Plots. Etc. Or directly use the text as labels in the screenshots. But if you do so, then the screenshot may become too clustered. In that case, maybe add an animation such that the labels show up and disappear one at a time.

**Create and share view page:** 
 - [x] Review the screenshots and make sure they are up-to-date. Eg, the Settings screenshot is old. Now it looks slightly different on Studio. So, replace it.
 - [x] When you say that in most cases you can skip these additional settings, elaborate a little further. You don’t have to provide full details here (coz you are linking to the Settings section). But still, give a little more information.
 - [x] Share a view should become a section of its own. In the future, if we have more things in the “Share” category (eg, suppose people can share specific plots), then they should all go this “share” section. This “share” section can become part of the “collaboration” section.
 - [x] GH app installation: This should be included in a separate page. Anything that is slightly advanced should be in a separate page. Don’t know yet what name to give to this section. Also, include more instructions as per feedback by Laurens.
 - [x] Add screenshots / instructions on how people can connect to GitLab and Bitbucket also.
 - [x] Explain that you can create multiple view per repo, and apply different settings to them.

**Configure a view page:**
 - [x] The starting paragraph is too wordy. Revise it.
 - [x] The section “Non-DVC repositories” should have a different heading. Eg, “Data source”. And the text within this section is too wordy. Revise it. The video within this does not really fit in the flow. Rewrite the flow slightly to make the video fit more naturally.
 - [x] Having a section “Configuring view settings” within the section “Configure a view” is odd. Within the sub-sections for the individual settings (eg, non-dvc repo, monorepo, data remotes, etc.), include the relevant part from the “Configuring view settings” so that you don’t need this separate section.
 - [x] View settings also have some more options now. Eg, View name. Revise this section and write about those options also. Also, replace the screenshot accordingly.
 - [x] Data remotes: Add some more instructions. Provide some more details on what kinds of remote storages can be used. If possible, create a short (30 second or so) video to show how this works. In fact, create a playlist with 4-5 short videos (around 30 seconds each) for each of the view settings (non-DVC repo, mono repo, data remotes, etc).
 
**Visualize and compare experiments page:**
 - [x] Make it more clear that the data that you are visualizing are simply csv, json etc. data. Maybe the video my Milecia fits more in this section if you write the overview accordingly.

**Run experiments page:**
 - [x] First, give a short overview of what it means to run experiments.
 - [x] Add some more details on how CML is used here. How are the cloud configurations defined? What other options do people have in addition to GitHub Actions for the CI/CD?.. Basically, figure out how CML works and then add a nice “Overview” section describing the experimentation use case.

**Teams page:**
 - [x] Present the “Roles” in a table/matrix format. Refer to https://www.notion.so/iterative/People-s-roles-in-a-team-workspace-af54fe6767c34eefba64085a7319eac3
 - [x] In the “Settings” section, explain how these settings are different from the view’s settings.

**New sections to add:**
 - [x] Main dashboard (where you see multiple views)
 - [x] Different options that people can use for the view display: Filters, Columns, Nested table headers, Selected only, Delta mode and save all these settings.
 - [x] User profile
 - [x] Troubleshooting

**Other tasks:**
 - [x] Confirm with Casper where the images will be stored and accordingly remove “_v2” from the image filenames.
 - [x] Find out why svg did not work in the banner image (probably was a bug in my code change #2540 )

**Comment by tapadipti** [comment]
@casperdcl The `static` repo (https://github.com/iterative/static/tree/main/img/studio) should continue to contain all the images for the Studio docs, right? I can see this in your earlier [comment](https://github.com/iterative/dvc.org/pull/2510#discussion_r643136007), so wanted to confirm before I try to figure out how to manage image updates as we update the docs in the future.

> we'd initially used static.iterative.ai because the images were WIP and we knew we'd change them
> now that they're finalised (I assume? #2469) we can put them in this repo as usual

**Comment by casperdcl** [comment]
> The `static` repo

I think the real issue is #1115 but until that's fixed

- if the images are "final" (i.e. unlikely to be changed) then I believe we can put them directly in this repo (not static)
- however (CC @shcheklein) if the "final" plan is to have a separate studo docs site (e.g. `dvc-studio.com/docs` or similar) then we should keep using static for now

Basically trying to avoid bloating repo history with unused binary images.

**Comment by tapadipti** [comment]
@casperdcl images are mostly final, except that they'll have to be updated every time the corresponding feature on Studio changes (there are already some that need changing - eg, View Settings). And yes, there's one more revision happening for the docs content, and as a result, some of the images may change and new images will be added. The problem with hosting images separately on a repo that is shared by production and dev is that if I need to update an image in dev, I end up updating it in prod also. There is no opportunity to review it before it reflects on the prod site, unless I give it a new name. That is why I used _v2 for all the images this time. If the plan is to have the images back in the docs repo eventually, then I'll do that in the next iteration of the docs.

**Comment by tapadipti** [comment]
I've compiled a list of required enhancements to the Studio docs [here](https://docs.google.com/document/d/1u9favvPTKdgZbbJhpRpGsN5x6ft9ZRiYnNaxSNztTwM/edit?usp=sharing). Pls review and and add/update as needed. Then, I'll create tasks or add them to this ticket as required.

**Comment by casperdcl** [comment]
> if I need to update an image in dev, I end up updating it in prod also

you can update URLs to point to a dev branch temporarily pre-merge

**Comment by tapadipti** [comment]
Decided to not implement the following, and have marked them as complete. Pls comment if you think these should be done:

- Share a view should become a section of its own.
- GH app installation: This should be included in a separate page. 
- Having a section “Configuring view settings” within the section “Configure a view” is odd

**Comment by tapadipti** [comment]
> I think the real issue is #1115 but until that's fixed
> 
> * if the images are "final" (i.e. unlikely to be changed) then I believe we can put them directly in this repo (not static)
> * however (CC @shcheklein) if the "final" plan is to have a separate studo docs site (e.g. `dvc-studio.com/docs` or similar) then we should keep using static for now
> 
> Basically trying to avoid bloating repo history with unused binary images.

I believe the plan for now is to host the images in the static repo. So, I'm leaving the images there. Pls let me know if that's not the plan @casperdcl @shcheklein
